packages/cloud-functions/package.json

```diff
@@ -23,7 +23,7 @@
     "bignumber.js": "^9.0.0",
     "fast-json-stable-stringify": "~2.1.0",
     "firebase-admin": "^9.5.0",
```

Comment:
In the description you mentioned you updated the firebase admin package. Did you also want to update that?

Reply:
Oops my bad...I meant to say `firebase-functions`! I've edited the description